imaginary/eimaginary.py

```diff
@@ -22,23 +22,24 @@ class NoSuchCommand(ImaginaryError):
 
 class AmbiguousArgument(ImaginaryError):
     """
-    One or more of the inputs specified can not be narrowed down to
-    just one thing.  This can be due to the presence of multiple
-    things with similar names, or due to the absence of anything named
-    similarly to the given input.
+    One or more of the inputs specified can not be narrowed down to just one
+    thing.  This can be due to the presence of multiple things with similar
+    names, or due to the absence of anything named similarly to the given
+    input.
 
     @ivar action: The action which was being processed when an ambiguity was
-    found.
+        found.
```

Comment:
While it should perhaps have a `@type`, I don't think that this should block this PR.